### PR TITLE
🌐 Add Chinese translation for `docs/zh/docs/advanced/openapi-webhooks.md`

### PR DESCRIPTION
**Pull request by Zhongheng-Cheng**
Add Chinese translation for `docs/zh/docs/advanced/openapi-webhooks.md`.
给 `docs/zh/docs/advanced/openapi-webhooks.md` 增加了中文翻译。

### CODE REVIEW

File: docs/zh/docs/advanced/openapi-webhooks.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+`app.webhooks` 对象实际上只是一个 `APIRouter` ，与您在使用多个文件来构建应用程序时所使用的类型相同。
+
+///
+
+请注意，使用网络钩子时，您实际上并没有声明一个*路径*（比如 `/items/` ），您传递的文本只是这个网络钩子的**标识符**（事件的名称）。例如在 `@app.webhooks.post("new-subscription")` 中，网络钩子的名称是 `new-subscription` 。
+
+这是因为我们预计**您的用户**会以其他方式（例如通过网页仪表板）来定义他们希望接收网络钩子的请求的实际 **URL 路径**。
+
+### 检查文档
```
Comment (Vincy1230) [inline docs/zh/docs/advanced/openapi-webhooks.md:49]:
```suggestion
### 查看文档
```

**Review by Vincy1230 — CHANGES_REQUESTED** [review]
辛苦了！这是一些修改建议，供您参考！

**Comment by Zhongheng-Cheng** [comment]
> 辛苦了！这是一些修改建议，供您参考！

Thanks! Looks better now!

**Review by Vincy1230 — APPROVED** [review]
LGTM :)

**Review by under-tree — APPROVED** [review]

**Comment by alejsdev** [comment]
Thank you! @Zhongheng-Cheng :rocket: 
And thanks for your review @Vincy1230 :100:

**Pull request merged**